observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -453,3 +453,108 @@ processors:
 ----
 <1> Specifies attribute keys to group by.
 <2> If a processed span, log record, or metric datapoint contains at least one of the specified attribute keys, it is reassigned to a Resource that shares the same attribute values; and if no such Resource exists, a new one is created. If none of the specified attribute keys is present in the processed span, log record, or metric datapoint, then it remains associated with its current Resource. Multiple instances of the same Resource are consolidated.
+
+[id="transform-processor_{context}"]
+== Transform Processor
```

Comment:
Nothing to do about it now, but in the future I'd recommend that these each be modules instead of all of the content being directly in the assembly if possible.

For example, you might want to consider alphabetizing this list of processors. But if you did, it would be a lot more copy/paste and look way different when reviewing the PR - as opposed to just changing on single lines of module inclusions to do the reorder.

Again, nothing to change now, just wanted to toss out the suggestion!